tools/Google.Cloud.Tools.Analyzers.Tests/Google.Cloud.Tools.Analyzers.Tests.csproj

```diff
@@ -0,0 +1,183 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Any reason this is an "old-style" project instead of a new one? I much prefer the new format, and then we don't need to worry about nuget restore etc.